src/dex/uniswap-v2/uniswap-v2-e2e-mainnet.test.ts

```diff
@@ -0,0 +1,1333 @@
+import dotenv from 'dotenv';
+dotenv.config();
+
+import { testE2E } from '../../../tests/utils-e2e';
+import { Tokens, Holders } from '../../../tests/constants-e2e';
+import {
+  Network,
+  ProviderURL,
+  ContractMethod,
+  SwapSide,
+} from '../../constants';
+import { JsonRpcProvider } from '@ethersproject/providers';
+
+describe('UniswapV2 E2E Mainnet', () => {
+  const network = Network.MAINNET;
+  const tokens = Tokens[network];
+  const holders = Holders[network];
+  const provider = new JsonRpcProvider(ProviderURL[network]);
+
+  it('50WETH -> ALPHA', async () => {
+    await testE2E(
+      tokens.WETH,
+      tokens.ALPHA,
+      holders.WETH,
+      '50000000000000000000',
+      SwapSide.SELL,
+      undefined,
+      undefined,
+      network,
+      provider,
+    );
+  });
+
+  describe('UniswapV2', () => {
+    const dexKey = 'UniswapV2';
+    const network = Network.MAINNET;
+    const tokens = Tokens[Network.MAINNET];
+    const holders = Holders[Network.MAINNET];
+    const provider = new JsonRpcProvider(ProviderURL[network]);
```

Comment:
don't need these 4 lines, the values are already set above...

Reply:
Fixed